src/seedu/addressbook/data/person/Address.java

```diff
@@ -1,58 +1,145 @@
 package seedu.addressbook.data.person;
 
 import seedu.addressbook.data.exception.IllegalValueException;
+import java.util.*;
```

Comment:
This is ok for internal imports, but should be avoided for external libraries, because you can be importing a huge namespace.